run_sim.py

```diff
@@ -204,57 +235,59 @@ def cli():
     show_default=True,
     help="Skip or rerun if the simulation output already exists",
 )
-@click.option(
-    "--make-missing-obsparam/--skip-missing-obsparam",
-    default=True,
-    show_default=True,
-    help="Make the obsparam or skip running the simulation if the obsparam is missing",
-)
 @click.option(
     "--force-remake-obsparams",
     is_flag=True,
     help="If set, remake all obsparams before running the simulations",
 )
 @click.option(
     "--log-level",
-    default="WARNING",
+    default="INFO",
     type=click.Choice(["INFO", "WARNING", "ERROR", "DEBUG"], case_sensitive=True),
     show_default=True,
     help="Verbosity level, also pass the flag to hera-sim-vis.py",
 )
 @click.option("-d", "--dry-run", is_flag=True, help="Pass the flag to hera-sim-vis.py")
-def h4c_cli(
+def hera_cli(
+    layout,
+    ants,
     freq_range,
     freqs,
     sky_model,
-    chunks,
+    n_time_chunks,
+    do_time_chunks,
     gpu,
-    hpc_config,
     slurm_override,
     skip_existing,
-    make_missing_obsparam,
     force_remake_obsparams,
     log_level,
     dry_run,
 ):
-    """Run H4C validation simulations.
+    """Run HERA validation simulations.
 
-    Use the default parameters, configuration files, and directories for H4C sims
+    Use the default parameters, configuration files, and directories for HERA sims
     (see make_obsparams.py).
     """
-    run_h4c_sim(
-        freq_range,
-        freqs,
-        sky_model,
-        chunks,
-        gpu,
-        hpc_config,
-        slurm_override,
-        skip_existing,
-        make_missing_obsparam,
-        force_remake_obsparams,
-        log_level,
-        dry_run,
+    if layout is not None and ants is not None:
+        raise ValueError("Provide only layout or ants")
+    if layout is None and ants is None:
+        raise ValueError("Either layout or ants must be provided")
+    if ants is not None:
+        layout = list(ants)
+
+    run_validation_sim(
+        layout=layout,
+        freq_range=freq_range,
+        freqs=freqs,
+        sky_model=sky_model,
+        chunks=n_time_chunks,
+        do_time_chunks=do_time_chunks,
+        gpu=gpu,
+        slurm_override=slurm_override,
+        skip_existing=skip_existing,
+        force_remake_obsparams=force_remake_obsparams,
+        log_level=log_level,
+        dry_run=dry_run,
     )
 
 
```

Comment:
Same as in `make_sky_model.py`, we should probably get rid of the "custom" cli and command now that they are effectively replaced by the `layout` option.

Reply:
I removed the `custom` command because it does nothing -- but I've still left the `hera` command as a sub-command because it's possible that in the future we want to test on things other than hera.